app/src/main/res/values/colors.xml

```diff
@@ -17,7 +17,7 @@
 
     <color name="onboarding_button_higlight_stroke">@color/photonBlue50</color>
 
-    <color name="browser_overlay_background">@color/photonGrey70</color>
+    <color name="browser_overlay_background">#f21C1B22</color>
```

Comment:
How would the alpha affect performance?

Reply:
In Android, when you draw one view on top of another, you get overdraw* and suffer a performance penalty. When one of those views contains alpha, it's worse: you suffer the same performance penalties as overdraw _and_ you have to compute the new blended value for the pixel (side note: I believe this combining two views where one has alpha is called "compositing"). In general, adding alpha values is a red flag that performance will be affected negatively.

While doing this computation once or a handful of times is fine, afaik, when the WebView is loading pages, it redraws the page over and over again as new elements are loaded. When you combine this with the 60 FPS redraws from animations while moving through items on the homescreen, you're more likely to start dropping frames.

In theory, it's possible this also slows down page load: since one CPU core is more committed to redrawing the page, it can't be used for page load.

I didn't measure frames dropped objectively so I don't know for sure but it felt like we were dropping frames so I wanted to split it out so we can investigate further and see if the trade-off is worthwhile.

\*overdraw is when you draw one color in a pixel and replace it with another color later, invalidating the need for the first pixel to have ever been drawn. If you do this for many pixels (or many times for single pixels), it non-trivially slows down drawing and you can start dropping frames.

Reply:
On the other hand, we have a huge problem with people not seeing the overlay as an overlay (because it... kinda isn't yet).  AFAIK, our major proposals for fixing this are basically adding transparency or splitting the overlay itself.  Either one is going to mean rendering the WV in the background.  We should test, but I wouldn't expect a perceivable perf difference between the two.

TL;DR my guess-that-is-backed-by-zero-data that the perf difference between split/transparent overlay is minimal.  If so, we might need to take one way or another in order to improve our UX.

Reply:
> we have a huge problem with people not seeing the overlay as an overlay

But is this solved by animating the overlay in?

Reply:
I think animating it in helps with this. That way it is easier to tell where it is in relation to the webpage. By fading in, it is clearer that it is a page that is just over the webpage. The transparency will help even more.